### PR TITLE
add NIH and course updates

### PR DESCRIPTION
**Pull request by vortexing**
This PR is primarily for the Data Science/DaSL section of the Wiki to start building some basic visibility into some of the early efforts we've undertaken that are Hutch oriented.  

Some of this content is borrowed from the course being made, and it links out to our blog post about drafts (so that we don't have to update the Wiki every 5 seconds).  

This could be a future home for final drafts as well so they are full-text searchable in the Wiki for additional find-ability.

### CODE REVIEW

File: _datascience/data_sharing.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+- **Supports transparency** - Sharing data provides more clarity about how studies are performed. Many scientists also believe in an ethical responsibility to study participants [@bauchner_citation_2016].
+- **Encourages reproducibility and rigor** - Having the data accessible, allows others to try to reproduce study findings. This can further enable studies that may replicate or validate the initial findings with different data. 
+- **Supports multi-modal work** - When more data of various types are easily available it makes it easier for scientist to perform studies with multiple types of data [@thessen_citation_2021]. 
+- **More efficient and cost effective** - Some data are especially difficult or expensive to produce.
+- **Supports Researcher Inclusion** - Data generation can be especially difficult for those at institutes with less resources. Publicly available data can therefore be used by these researchers to better enable their participation. 
+- **Increased impact** - Papers that share their data in repositories appear to be cited more based on the study by @colavizza_citation_2020 .
```
Comment (vortexing) [inline _datascience/data_sharing.md:20]: oop, I need to remove/adjust these citations that won't be rendered right here.

File: _datascience/hutch_courses.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+The Fred Hutch Data Science Lab's wider training efforts can be found on our [training page](https://hutchdatascience.org/training/).  However, in addition to data science training courses we create that are focused on a wider audience, we have also created a number of Fred Hutch oriented training materials.  
+
+
+## Cluster 101
+We collaborated with SciComp and created a course called [Cluster 101](https://leanpub.com/courses/fredhutch/fredhutchcluster101).  This course can be taken anytime, anywhere, for free (don't worry just set the LeanPub cost to zero) and it’ll help you check to make sure your account is set up to use the cluster and either help you check to make sure you understand the basics of how to use a cluster, or help you get up to speed.  It can even give you a certification for reporting between staff and lab leaders if need be.  
+
+If you take this course and want to give us feedback or would like to learn more about it, you can share your thoughts in [Slack in the #dasl-news channel](https://fhdata.slack.com/archives/C03T4QPNDA9). Or you can file an issue on [the course's GitHub repository](https://github.com/fhdsl/FH_Cluster_Guide).
```
Comment (vortexing) [inline _datascience/hutch_courses.md:12]: I should adjust these to be the #ask-dasl channel (link is here: https://fhdata.slack.com/archives/C043ZVCBY9Z)

**Comment by vortexing** [comment]
Something is angry in the build (I suspect navbar yaml spaces but don't know yet).  

BUT, I'd like to have this page replace this old page in Data Gen:
https://sciwiki.fredhutch.org/generation/human_shareDeposits/

It's markdown is here: 
https://github.com/FredHutch/wiki/blob/main/_generation/human_shareDeposits.md

Last time we did get folks in the IRO to review the content and that might be something that would be of interest again but I wouldn't want to hold this up for that.  I'd like to remove the page in Data Gen and find any links that would be broken as part of us deprecating the content in that section.

**Comment by dtenenba** [comment]
The build system says:

```
jekyll 3.9.2 | Error:  (/srv/jekyll/_data/navigation.yml): did not find expected key while parsing a block mapping at line 57 column 5
```

You can try an online yaml parser to debug this.

File: _data/navigation.yml [hunk 3]
```diff
@@ -50,10 +50,18 @@ datascience:
         url: /datascience/dc_index/
       - title: "Pathways"
         url: /pathways/
-      - title: "FH-DaSL Courses"
+  - title: "Data Management"
+    children:
+      - title: "NIH Data Sharing"
+        url: /data_sharing/
+  - title: "Training"
```
Comment (atombaby) [inline _data/navigation.yml:57]: I think it's angry 'cos this is missing a `children:` whatsit:
```
 57   - title: "Training"
 58     children:
 59       - title: "DaSL Courses for the Hutch"
 60         url: /hutch_courses/
 61       - title: "All DaSL Courses"
 62         url: "https://hutchdatascience.org/training/"
```

File: _datascience/data_sharing.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
+---
+title: NIH Data Sharing
+primary_reviewers: vortexing
+---
+
+## New Data Sharing Policy
+> What you should know about the new 2023 NIH Data Management and Sharing Policy
+
+Starting January 25th, 2023, the NIH will implement a new policy that requires new grant applications or renewals that generate scientific data to include a detailed data management and sharing plan (DMSP). The NIH expects all shareable data to be made available, whether it is associated with a publication or not. The DMS Policy is similar to what other funding agencies call a data management plan (DMP). The goals of this new policy are to establish the expectation for the management and sharing of scientific data generated from NIH-funded or conducted research, and to emphasize the importance of good data management practices.
```
Comment (atombaby) [inline _datascience/data_sharing.md:9]: Are these two phrases referencing the same thing?

> ... to include a detailed data management and sharing plan (DMSP) ...

and

> ... The DMS Policy is similar to what ...

Should the second one be "DMSP Policy"?  Or is "DMS Policy" a different plan/policy?

File: _datascience/data_sharing.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+- **Encourages reproducibility and rigor** - Having the data accessible, allows others to try to reproduce study findings. This can further enable studies that may replicate or validate the initial findings with different data. 
+- **Supports multi-modal work** - When more data of various types are easily available it makes it easier for scientist to perform studies with multiple types of data [@thessen_citation_2021]. 
+- **More efficient and cost effective** - Some data are especially difficult or expensive to produce.
+- **Supports Researcher Inclusion** - Data generation can be especially difficult for those at institutes with less resources. Publicly available data can therefore be used by these researchers to better enable their participation. 
+- **Increased impact** - Papers that share their data in repositories appear to be cited more based on the study by @colavizza_citation_2020 .
+- **Increased collaboration opportunity** - Having data available can encourage other researchers to expand the research in a new direction or extend it further and they may reach out to collaborate.
```
Comment (atombaby) [inline _datascience/data_sharing.md:21]: run on?

File: _datascience/data_sharing.md (hunk 4 above)
Comment (atombaby) [reply _datascience/data_sharing.md:9]: Or are they both different?  A DMS Plan and the DMS Policy?  DMS Policy is referenced down doc, and "DMSP" is never used outside this introduction.  Maybe change the first reference to:

>    ... to include a detailed data management and sharing plan (DMS plan) ...

**Review by atombaby — APPROVED** [review]

File: _data/navigation.yml (hunk 3 above)
Comment (dtenenba) [reply _data/navigation.yml:57]: Just committed what I think will be a fix for this. At least the online yaml validator says so.

Comment (vortexing) [reply _data/navigation.yml:59]: Thanks all. ;)

File: _datascience/data_sharing.md (hunk 4 above)
Comment (vortexing) [reply _datascience/data_sharing.md]: Good catch.  Yeah, the DMS policy requires a DMS plan. ;) I'll disambiguate!!!

File: _datascience/data_sharing.md (hunk 5 above)
Comment (vortexing) [reply _datascience/data_sharing.md]: yup, removed the middle!

**Comment by vortexing** [comment]
I'm going to merge this in and then we can keep iterating over them in the future.

**Pull request merged**